docker-compose/config/common.libsonnet

```diff
@@ -36,5 +36,5 @@
   browserUrl: 'http://localhost:7984',
   httpListenAddress: ':80',
   maximumMessageSizeBytes: 16 * 1024 * 1024,
-  global: { diagnosticsHttpListenAddress: ':9980' },
```

Comment:
Instead of removing this option, could you set it as follows?

https://github.com/buildbarn/bb-clientd/blob/master/configs/bb_clientd.jsonnet#L178

That way we can still do pod health checks and gather Prometheus metrics.